docs/Build.md

```diff
@@ -90,6 +90,8 @@ We also make use of the following plugins:
 | html-webpack-plugin      | Handles templating of built output into a provided index.html file |
 | mini-css-extract-plugin      | Compresses and emits a css file(s) containing all styling for the UI |
 | compression-webpack-plugin      | Compresses built output further using `gzip` algorithm, and emits these compressed files. Depending on headers provided by the browser, these gziped versions will be served to the browser, rather than the uncompressed versions |
+| webpack-bundle-analyzer      | At build time, produce a report regarding the JS bundle size (useful for understanding bloat and duplication). At dev build time this is an html file (which is then hosted by `webpack-dev-server`), and a json file at production build time. Each report is written to the `generated/bundle-analyser` directory |
+| BannerPlugin      | Used to add a copyright header to built css code. Other types handled by other plugins (eg TerserPlugin) |
```

Comment:
Any way we can get headers auto added to source?

Reply:
Yes - its in the linting implementation I have ready :)